src/main/java/design/MazeDesignerController.java

```diff
@@ -0,0 +1,52 @@
+package design;
+
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+
+public class MazeDesignerController {
+    MazeDesignerInteractor md = new MazeDesignerInteractor();
```

Comment:
From my understanding, we don't want to create instances of the interactor (or input boundary) or the presenter directly in our controller. 

Instead, we should add an instance attribute to store an instance of the interactor (or input boundary because you have one):

`private MazeDesignerInteractor md;`

But since you have an input boundary, it would be better if you did: 

`private MazeDesignerInputBoundary md;`

Then in your constructor for the controller: 

public MazeDesignerController(MazeDesignerInputBoundary md) {
    this.md = md;
}